lightning-limitations/README.md

```diff
@@ -16,15 +16,19 @@
 ## Discussion Questions
 
 ### Limitations of lightweight clients
+
 1. As a service, why is it easier to send funds than receive them?
 
 ### Fee Management
-2. Why would the fundee care about the size the fees if it’s a funder that’s paying the fees?
-3. If you disagree about the fees with your peer, how do you negotiate them?
+
+1. Why would the fundee care about the size the fees if it’s a funder that’s paying the fees?
+1. If you disagree about the fees with your peer, how do you negotiate them?
 
 ### Liquidity/Capacity
-4. What’s to prevent an attacker from temporarily drying up liquidity in the network, by sending payments to himself using really long routes, but does not complete the payments? by using X amount of funds, he can potentially lock up an amount several times of X, depending on how long he wants to construct the routes. This attack would not cost the attacker any money, only time. It would not be possible to attribute fault to the attacker as LN is onion-routed, so the routing nodes would have no idea where exactly the money is held up. The routing nodes could also be doing this attack among themselves, to compete for market share.
+
+1. What’s to prevent an attacker from temporarily drying up liquidity in the network, by sending payments to himself using really long routes, but does not complete the payments? by using X amount of funds, he can potentially lock up an amount several times of X, depending on how long he wants to construct the routes. This attack would not cost the attacker any money, only time. It would not be possible to attribute fault to the attacker as LN is onion-routed, so the routing nodes would have no idea where exactly the money is held up. The routing nodes could also be doing this attack among themselves, to compete for market share.
```

Comment:
What’s to prevent an attacker from temporarily drying up liquidity in the network, by sending payments to himself using really long routes, but does not complete the payments? This is a confusing sentence. Is it accurate if changed to -- What's to prevent an attacker from temporarily drying up network liquidity by sending himself payments using really long routes without completing the payments?